custom_connectors/basic_auth/transmit_sms_connector.rb

```diff
@@ -115,6 +115,17 @@
             control_type: "text-area",
             optional: false
           },
+          {
+            name: "tracked_link_url",
+            label: "Tracked Link",
+            type: "string",
+            control_type: "url",
+            optional: true,
+            hint: "Full URL that will be shortened and inserted via this "\
```

Comment:
@codechito For hint's and labels the text should be easily understood by the user, can this be rephrased as:
space missing between 'Link Hit` and trigger.

Are you referring to labels or `API names`?? For the end user, label make sense...

Also, I do not see `Link Hit` trigger in the Connector. Does it make sense to refer in the hint.

```
Full url that will be shortened and inserted via this **Tracked link** variable added to your **Message body** above. Hits to this link can be tracked and also passed via the 'Link hit' trigger. url should be in the format e.g. `https://www.mydomain.com/`.
```

Reply:
already added a space and about the link hit trigger it will submitted on a separate PR once done. we just received update that static and dynamic webhook can coexist in 1 connector so we didn't manage to included it in these PR. rest assured that these trigger mention will also available. probably this week or next week a new PR for the trigger will be pushed.